python/lsst/cp/verify/verifyStats.py

```diff
@@ -176,14 +195,16 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.makeSubtask("repair")
 
-    def run(self, inputExp, camera):
+    def run(self, inputExp, camera, taskMetadata=None):
         """Calculate quality statistics and verify they meet the requirements
         for a calibration.
 
         Parameters
         ----------
         inputExp : `lsst.afw.image.Exposure`
             The ISR processed exposure to be measured.
+        taskMetadata : `lsst.daf.base.PropertySet`, optional
```

Comment:
Is `RESIDUAL STDEV` the only statistic from the metadata that we currently use? Are there any other statistics in the metadata that could be useful?  Where would we need to look if we wanted to see all the contents of this metadata object to see if there's something else that might be useful?

Reply:
It's produced by `IsrTask`, and contains a variety of per-amp measurements at different stages of processing.  It also holds some statistics about the overscan.  It's stored there because when I reworked the task, I wanted to save some of the information that we were measuring that wasn't being saved anywhere.  This gets back to the design decision comment above; it's probably worth thinking about if this information should be written to the exposure metadata as well.

Reply:
Maybe we can discuss this more in one of our meetings, especially if we decide that we want to use these statistics as stability metrics or something like that.  In the end, I'm just concerned about 1) are we saving potentially useful information in this metadata, 2) if so, how can we easily access it for plotting/use/further analysis?